index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUSuario = prompt('Ingrese su nombre');
+alert(`Hola ${nombreUSuario}`);
```

Comment:
(Ej. de corrección, como una de las características de GitHub) 
En este caso, me parece mejor que no uses interpolación de strings.
(Ej. de sugerencia que se puede hacer, como otra característica de GitHub)
```suggestion
alert('Hola' + nombreUsuario );
```